build.gradle

```diff
@@ -17,7 +17,7 @@
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = '1.6.20'
+    ext.kotlin_version = '1.6.10'
```

Comment:
I admire your dedication but I don't want to use compose in the application yet. If I do the transition, I will migrate the whole app to avoid maintaining two systems.